chatterbot/adapters/plugins/evaluate_mathematically.py

```diff
@@ -0,0 +1,200 @@
+
+        # Returning important information
+        try:
+            string += '= ' + str( eval( string ) )#self.evaluate( string ) )
+
+            return string
+        except:
+            return False
+
+
+    def simplify_chunks(self, input_text):
+        """
+        Separates the incoming text.
+        """
+
+        expression = []
+        string = ''
+
+        for chunk in input_text.split( ' ' ):
+
+            is_integer = self.isInteger( chunk )
+
+            if is_integer == False:
+                is_float = self.isFloat( chunk )
+
+                if is_float == False:
+                    is_operator = self.isOperator( chunk )
+
+                    if is_operator == False:
+                        continue
+                    else:
+                        expression.append( is_operator )
+
+                        string += str( is_operator ) + ' '
+                else:
+                    expression.append( is_float )
+
+                    string += str( is_float ) + ' '
+            else:
+                expression.append( is_integer )
+
+                string += str( is_integer ) + ' '
+
+        return expression, string
+
+
+    def evaluate( self, expression ):
+        """
+        Evaluates a set of expressions
+        and produces an answer. Then,
+        it returns the answer.
+        """
+
+        return eval( expression )
+
+
+    def isFloat(self, string):
+        """
+        If the string is a float, returns
+        the float of the string. Otherwise,
+        it returns False.
+        """
+
+        try:
+            float( integer )
+
+            return float( integer );
+        except:
+            return False
+
+
+    def isInteger(self, string):
+        """
+        If the string is an integer, returns
+        the int of the string. Otherwise,
+        it returns False.
+        """
+
+        if string.isdigit():
+            return int( string )
+        else:
+            return False
+
+
+    def isOperator(self, string):
+        """
+        If the string is an operator, returns
+        said operator. Otherwise, it returns
+        false.
+        """
+
+        if string in "+-/*^\(\)":
+            return string
+        else:
+            return False
+
+
+    def normalize(self, string):
+        """
+        Normalizes input text, reducing errors
+        and improper calculations.
+        """
+
+        # Setting all words to lowercase
+        string = string.lower()
+
+        # Removing punctuation
+        string = re.sub( '[.!?/:;]', '', string )
+
+        # Removing words
+        string = self.substitute_words( string )
+
+        # Returning normalized text
+        return string
+
+    def load_data( self, language ):
+        """
+        Load language-specific data
+        """
+
+        if language == "english":
+            with open(os.path.join(os.path.dirname(__file__), 'data') + "/math_words_EN.json") as data_file:
```

Comment:
Instead of forcing `/` as the directory separator here, you can add `math_words_EN.json` as a third parameter to the `os.path.join` call which is already being used here.